.expeditor/deploy.pipeline.yml

```diff
@@ -21,6 +21,20 @@ steps:
 
   - wait
 
+  - command: .expeditor/buildkite/URL-verification.sh
+    label: ":expeditor:"
+    concurrency: 1
+    concurrency_group: chef-omnitruck-master/deploy/$ENVIRONMENT
+    expeditor:
+      executor:
+        docker:
+      secrets:
+        FASTLY_API_TOKEN:
+          path: account/static/fastly/eng-services-ops
+          field: token
+        
+  - wait
+
```

Comment:
This job should come _after_ we purge Fastly...you also don't need to pass the job `FASTLY_API_TOKEN`.